rbac/api/cross_access/model.py

```diff
@@ -56,9 +56,15 @@ def validate_input_value(self):
         if (
             isinstance(self.end_date, datetime.datetime)
             and isinstance(self.start_date, datetime.datetime)
-            and self.start_date.date() > self.end_date.date()
         ):
-            raise ValidationError("Start date must be earlier than end date.")
+            if self.start_date.date() > (datetime.datetime.now() + datetime.timedelta(60)).date():
```

Comment:
Good call on moving all the validation into the model.

Reply:
Thanks! Cleaned up a lot and made life a bit easier. May be worth checking which of the other validations we can move in as well.